_posts/2022-10-10-release.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Release 2022.40
+author: Priya
+date: 2022-10-10
+category: Jekyll
+layout: post
+---
+### Enhancements
+
+#### Updated navigation bar
+Implemented a new look and feel for the data library nav bar and side bar.The side bar on the landing page now includes two sections: "MyFiles" and "Report", with additional items to be added in the future.The "Add new data" option shifted to the right. Use the + icon on the top-right to fetch data from desktop, URLs, webhooks, and APIs & Databases..
+
+
+### Defect fixes
+
+* Resolved issue where preview was not showing up for the left side dataset's preview in Join rule.
+* Addressed UI distortion on the Query box screen for Third party model.
+* Fixed an issue where the popover was hidden behind the data grid when the live link was generated in Live link.
+* Join rule- Editing join rule was not showing the correct number of rows in preview at that point in pipeline.Fixed this issue.
+* Resolved typo in Plan cards for checkout button in Settings-Plan.
+* Resolved issue where clicking on the mammoth icon on the setting page resulted in an empty landing page.
+* Addressed issue where toast notifications for adding more data and third party window appeared in the background.
+* Resolved issue where there was no proper error message while creating a share-point connection.
+* Resolved issue where the action resulted in an "oops something went wrong" notification instead of showing a proper error message.
+* Resolved issue where deleting a foreign view in Join rule showed false deletion.
+* Column browser had UI and function issues.
```

Comment:
Make this a positive statement. We can also remove this sentence, if the issue is not worth mentioning